README.md

```diff
@@ -1016,10 +1285,17 @@ const closeConnection = conn => Future((rej, res) => {
 ```
 
 #### finally
-##### `#finally :: Future a b ~> Future a c -> Future a b`
-##### `#lastly :: Future a b ~> Future a c -> Future a b`
-##### `.finally :: Future a c -> Future a b -> Future a b`
-##### `.lastly :: Future a c -> Future a b -> Future a b`
+
+<details><summary><code>finally :: Future a b ~> Future a c -> Future a b</code></summary>
+
+```hs
+finally :: Future a b ~> Future a c               -> Future a b
+lastly  :: Future a b ~> Future a c               -> Future a b
+finally ::               Future a c -> Future a b -> Future a b
+lastly  ::               Future a c -> Future a b -> Future a b
```

Comment:
Alternatively, I can go with the Sanctuary approach of naming the function exactly after where it lives, so `prototype.finally`, for example. Can you look at other functions (like `fork`) and see if those are also confusing in their current form?